### ISSUE TITLE
@desc tag not work

### ISSUE THREAD

**Issue by hughxusu** [Kocal/jsdoc-vuejs#140]
```javascript
<script>
/**
 * @desc components detail XXXXX
 * @vue-data {Page} page - XXX
 * @vue-data {Layer} layer - XXX
 */
export default {
  name: 'Panel',
  data() {
    return {
      page: null,
      layer: null
    };
  }
};
</script>
```

content in @desc not output in docs, how to handle it? Thanks~~~

**Comment by Kocal** [comment]
That's because of this line: https://github.com/Kocal/jsdoc-vuejs/blob/master/index.js#L61

Change the line to `e.doclet.description = (e.doclet.description || '') + str;` make `@desc` works.

I'm working on a patch.

**Comment by hughxusu** [comment]
it work, thank you~~~

**Comment by Kocal** [comment]
Released in v2.3.2
